spiff-arena-common/src/spiff_arena_common/runner.py

```
@@ -122,18 +131,36 @@ def __init__(self):
             "time": time,
             "timedelta": datetime.timedelta,
         })
+        self.custom_scripts = {}
+
+    def load_custom_scripts(self, path):
+        self.custom_scripts = {}
+        try:
+            with open(path) as f:
+                source = f.read()
+            namespace = {}
+            exec(source, namespace)
```

Ruff (0.15.10)

[error] Use of `exec` detected

(S102)